src/cmds/qstat.c

```diff
@@ -942,11 +983,21 @@ altdsp_statjob(struct batch_status *pstat, struct batch_status *prtheader, int a
 				timeval);
 		} else {
 			if(how_opt & ALT_DISPLAY_INCR_WIDTH) {
+				if (strlen(pstat->name) > 20) {
```

Comment:
Should this be jname instead of pstat->name?

Reply:
Same as above, jname hasn't been filled yet.